package.json

```diff
@@ -111,17 +111,17 @@
       },
       {
         "command": "PowerShell.OpenInISE",
-        "title": "Open current file in PowerShell ISE",
+        "title": "Open Current file in PowerShell ISE",
```

Comment:
Did you edit this online or via a local fork? If a fork then `Add more commits by pushing to the develop branch on SchneiderAndy/vscode-powershell.`